tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
> My point that abstract identifiers are more harm/confusion than benefit. Maybe on Debian something is in one package but in another distribution you need two packages (for whatever reason, maybe the base is different). Trying to cram this into a ever-growing table that one must look up each time a new test is written feels wrong to me. 

Actually that is what you normally do in programming. You fold those intends into a well defined interface like "I want to print a document". Everything necessary for this isn't handled by your call to the interface but by the implementation below. By putting all these tiny details into the test cases we only make those unnecessarily complex with a lot of distribution specific things.

If there is a much more complex setup required for specific things we could even spell those things out into individual functions later on like `distro_install_and_configure_cups`. However for what we have today those mapping tables are not that huge and I don't see them getting them real huge in the near future.